slowly-changing-lookup-cache/bin/deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Do you need this file? You already mentioned the procedure to follow in README.md. 
I find this shell script a bit redundant.

Reply:
Indeed. This will be removed next commit.